ibis/backends/flink/registry.py

```diff
@@ -50,7 +50,8 @@ def _cast(translator: ExprTranslator, op: ops.generic.Cast) -> str:
             return f"TO_TIMESTAMP(CONVERT_TZ(CAST({arg_translated} AS STRING), 'UTC+0', '{to.timezone}'))"
         else:
             return f"TO_TIMESTAMP({arg_translated})"
-
+    elif to.is_date():
+        return f"CAST({arg_translated} AS date)"
```

Comment:
@mfatihaktas I removed "date" from `sql_type_mapping` and added this here; does it sound good for now?

Probably should look to make `TRY_CAST` consistent with the added logic here in the future, but I think it's not a priority immediately.

Reply:
I think it looks good. Thanks!